<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Orbiter Addons Manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/res/icon/satellite.ico</normaloff>:/res/icon/satellite.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="nameLabel">
      <property name="text">
       <string>Orbiter Addons Manager</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="listsLayout">
      <item>
       <widget class="QGroupBox" name="enabledGroup">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Enabled add-ons</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QListWidget" name="enabledAddonsList">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="statusTip">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="disableButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="statusTip">
            <string>Disable the selected add-on</string>
           </property>
           <property name="text">
            <string>Disable</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="disabledGroup">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Disabled add-ons</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QListWidget" name="disabledAddonsList">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="enableButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="statusTip">
            <string>Enable the selected add-on</string>
           </property>
           <property name="text">
            <string>Enable</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonsLayout">
      <item>
       <widget class="QPushButton" name="installButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="statusTip">
         <string>Install an add-on</string>
        </property>
        <property name="text">
         <string>Install</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="uninstallButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="statusTip">
         <string>Uninstall the selected add-on</string>
        </property>
        <property name="text">
         <string>Uninstall</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSettings"/>
   <addaction name="actionRescan"/>
   <addaction name="actionAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About Orbiter Addons Manager</string>
   </property>
   <property name="statusTip">
    <string>About Orbiter Addons Manager</string>
   </property>
  </action>
  <action name="actionRescan">
   <property name="text">
    <string>Rescan add-ons</string>
   </property>
   <property name="iconText">
    <string>Rescan add-ons</string>
   </property>
   <property name="toolTip">
    <string>Rescan add-ons in Orbiter installation</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Open the settings window</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
